packages/components/typography/src/Heading.tsx

```diff
@@ -19,6 +21,8 @@ const HeadingTypes: Record<
   h4: Heading4,
   h5: Heading5,
   h6: Heading6,
+  heading7: Heading7,
```

Comment:
I think doesn't make sense to map a semantic element that doesn't exist in HTML. Why should we provide h7 and h8?

Reply:
https://www.figma.com/design/uA9V2FOWZSX31kI4wGVpWZ/Design-System-Handoff?node-id=1-166&node-type=canvas&m=dev asked by design